<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>311</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Plus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_abc">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>( )</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
